eva-lib/src/main/java/uk/ac/ebi/eva/lib/config/EvaProperty.java

```diff
@@ -0,0 +1,130 @@
+package uk.ac.ebi.eva.lib.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@ConfigurationProperties(prefix = "eva")
+@Component
+public class EvaProperty {
+
+//    eva.mongo.host=@eva.mongo.host@
```

Comment:
Please review these properties to make them as similar as possible to [those in eva-pipeline](https://github.com/EBIvariation/eva-pipeline/blob/develop/examples/application.properties).

For instance, it should be possible to use Spring Data MongoDB properties instead of our own fields. Further work to inject that automatically could be necessary and I guess could belong in another PR.